src/styles/_alpheios_backup.scss

```diff
@@ -0,0 +1,57 @@
+$alpheios-uikit-namespace: 'auk';
```

Comment:
Do we need a backup file? If its purpose is to save an old state of things, we have VCS for that

Reply:
oh, you are right - I will remove all old files.